packages/loader/container-utils/src/deltaManagerProxyBase.ts

```diff
@@ -22,6 +22,8 @@ import {
 /**
  * Base class for creating proxy to the real delta manager. It implements all required methods on IDeltaManager and
  * proxy implementations can override specific methods.
+ *
+ * @deprecated No replacement API is recommended.
```

Comment:
I'd do something similar here to what I suggest in the changeset. An FF dev with no context about this that runs into the class might interpret this as "this thing is completely disappearing" which is not exactly right.